### DISCUSSION TITLE
New XeSS DLL available

### DISCUSSION THREAD

**Discussion by AleksNo** [optiscaler/OptiScaler#110] [General]
Hello, 

Intel published a new XeSS DLL. You can download it here:
https://software.intel.com/sites/downloads/symbols/igxess2.dll/66E4C3AC6479000/

But i do not know what they improved. :)

**Comment by cdozdil** [comment]
Hi, 

Thanks for interest but that is not XeSS upscaling dll. Correct file is libxess.dll

**Reply by mikk9** [reply]
It is correct, you can rename it into libxess.dll (it's 1.3.91). But seems to work only on Arc cards and therefore not recommended.

**Reply by AleksNo** [reply]
Ohhh, thank you. :)

**Reply by tmrdev** [reply]
Does it include Frame Generation support? Does it only run on Vulcan?

**Reply by cdozdil** [reply]
There are a few other exports but not sure what they are for

**Reply by mikk9** [reply]
We should know more when Battlemage comes this month

**Comment by cdozdil** [comment]
Hi,

@AleksNo I haven't noticed but Maple checked that file and it exports Vulkan upscaling methods. 
I have checked and implemented it, so I can confirm Vulkan upscaling works on Arc cards.

![image](https://github.com/user-attachments/assets/8f094af3-534f-4223-aaae-128c0f3851f7)

Ignore the version number (not sure why but Opti couldn't read it from dll)

**Reply by juniorm-dev** [reply]
Hi,
Could you give quick step by step instructions on how you got it to work?
Currently getting incompatible driver error, for new Indiana Jones game.

**Reply by cdozdil** [reply]
Hi,

Sadly with latest Indiana Jones game OptiScaler is not working. (at least all reports were in that direction)

For Vulkan XeSS on non-Intel cards you need XeSS 2.0 files for F124 and build pinned at Discord support channel.

**Reply by juniorm-dev** [reply]
Thanks for quick reply.
So only the XeSS dp4a path works for Vulcan and not native xmx. That's pretty strange.

**Reply by cdozdil** [reply]
Ah no, for non-Intel cards only Vulkan works (Dx11 is no go). 
For Intel cards now XeSS supports Dx11, Dx12 and Vulkan.
